js/mexc3.js

```diff
@@ -12,6 +12,8 @@ const Precise = require ('./base/Precise');
 module.exports = class mexc3 extends Exchange {
     describe () {
         return this.deepExtend (super.describe (), {
+            'apiKey': 'mx05T6jNFFdDqvvHHt',
```

Comment:
I think you accidentally push the credential. You can put keys in `keys.local.json`.